packages/login/theme/lumo/vaadin-login-form-wrapper-styles.js

```diff
@@ -17,6 +17,11 @@ const loginFormWrapper = css`
     margin-top: calc(var(--lumo-font-size-xxxl) - var(--lumo-font-size-xxl));
   }
 
+  ::slotted([slot='submit']) {
+    margin-top: var(--lumo-space-l);
+    margin-bottom: var(--lumo-space-s);
```

Comment:
Here's a potentially breaking change: using `::slotted()` decreases specificity (so e.g. global app level CSS for `vaadin-button` will start to apply). However, that also applies to the `forgot-password` button.